### ISSUE TITLE
Question about the TAFT module

### ISSUE THREAD

**Issue by canglangzhige** [slei109/PATNet#16]
Thanks for your work!
A regression loss is used in TAFT for training the transformation matrix. But there is no such loss in your work.
Does the PATM module not need to be trained? Why?

**Comment by slei109** [comment]
PATM is trained. We adopt Cross_Entropy as the loss function and implement it in the compute_objective() function.
